### PR TITLE
Fix spelling errors

### PR DESCRIPTION
**Pull request by jsoref**
Issue:

## What I did

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/storybook/commit/705ed3e023abc6ab761700a1f218784850282040#commitcomment-42766948

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/storybook/commit/5b60fb5641fdb62768ce40bd5c6de764d05f048d

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

## How to test

- Is this testable with Jest or Chromatic screenshots?
no
- Does this need a new example in the kitchen sink apps?
no
- Does this need an update to the documentation?
no

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Comment by jsoref** [comment]
(I won't be looking at this PR until Tuesday)

File: .spelling [hunk 1]
```diff
@@ -71,7 +71,6 @@ storybook.js.org
 YuzuJS
 setImmediate
 Malte
-Ubl
```
Comment (jsoref) [inline .spelling:74]: I'm assuming that this is similar to a file I have, I can find no particular reason for this item to exist.

File: examples/official-storybook/stories/core/prefix.stories.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Very simple stories to show what happens when one story's id is a prefix of anothers'
+// Very simple stories to show what happens when one story's id is a prefix of another's
```
Comment (jsoref) [inline examples/official-storybook/stories/core/prefix.stories.js:1]: I'm only vaguely confident about this one.

File: lib/client-api/src/story_store.test.ts [hunk 3]
```diff
@@ -439,7 +439,7 @@ describe('preview.story_store', () => {
           },
           globalTypes: {
             arg2: { defaultValue: 'arg2' },
-            arg3: { defautlValue: { complex: { object: ['changed'] } } },
+            arg3: { defaultValue: { complex: { object: ['changed'] } } },
```
Comment (jsoref) [inline lib/client-api/src/story_store.test.ts:442]: This is suspicious

File: lib/components/src/tabs/tabs.tsx [hunk 4]
```diff
@@ -204,10 +204,10 @@ Tabs.displayName = 'Tabs';
   bordered: false,
 };
 
-type FuncChilden = () => void;
+type FuncChildren = () => void;
```
Comment (jsoref) [inline lib/components/src/tabs/tabs.tsx:207]: Technically `-en` is a plural marker, as is `-r`, but English is quirky, so we went with `children` not `childr` nor `childen`.

**Review by jsoref — COMMENTED** [review]
Note: This pr was generated fairly mechanically. I let Google Sheets make pretty much all of the suggestions, and I applied them. I tried fairly hard to avoid corrupting substring words, but I was in a rush, so it wouldn't shock me if I did.

And given the season, please accept my apologies if I did so.

File: lib/cli/src/generators/UPDATE_PACKAGE_ORGANIZATIONS/index.ts [hunk 5]
```diff
@@ -54,7 +54,7 @@ function updateSourceCode(parser: string) {
   const jscodeshiftPath = path.dirname(require.resolve('jscodeshift'));
   const jscodeshiftCommand = path.join(jscodeshiftPath, 'bin', 'jscodeshift.sh');
 
-  ['update-organisation-name.js', 'move-buildin-addons.js'].forEach((codemod) => {
+  ['update-organisation-name.js', 'move-builtin-addons.js'].forEach((codemod) => {
```
Comment (shilman) [inline lib/cli/src/generators/UPDATE_PACKAGE_ORGANIZATIONS/index.ts:57]: If you want to change this and the similar ones below, you need to rename the file too

**Review by shilman — CHANGES_REQUESTED** [review]
This is golden--thanks so much for doing it!

Please fix the issue below. You'll probably also need to update the snapshots since the tests are failing:

```
yarn test --core --update
```

File: lib/cli/src/generators/UPDATE_PACKAGE_ORGANIZATIONS/index.ts (hunk 5 above)
Comment (jsoref) [reply lib/cli/src/generators/UPDATE_PACKAGE_ORGANIZATIONS/index.ts:57]: It is renamed: https://github.com/storybookjs/storybook/pull/12590/files#diff-1f7e4ef72821f3812b9000a0a8918760. GitHub unfortunately makes it very easy to miss such renames. (I often fail to catch such renames, so I appreciate the call out.)

Comment (shilman) [reply lib/cli/src/generators/UPDATE_PACKAGE_ORGANIZATIONS/index.ts:57]: Hmm. Test is failing tho?

>     Cannot find module '/tmp/storybook/lib/codemod/src/transforms/move-buildin-addons' from 'lib/codemod/src/transforms/__tests__/transforms.tests.js'

Comment (jsoref) [reply lib/cli/src/generators/UPDATE_PACKAGE_ORGANIZATIONS/index.ts:57]: Oops, I renamed one file but not a related directory :-( fixing.

File: addons/docs/src/frameworks/angular/__testfixtures__/doc-button/compodoc.snapshot [hunk 6]
```diff
@@ -132,7 +132,7 @@ like <strong>bold</strong>, <em>italic</em>, and <code>inline code</code>.</p>
           "name": "click",
         },
       ],
-      "id": "component-InputComponent-8088cb533b6defa9d43d10841b59ce18",
+      "id": "component-InputComponent-14bbde487c28642f97f1f6c94b65ab31",
```
Comment (jsoref) [inline addons/docs/src/frameworks/angular/__testfixtures__/doc-button/compodoc.snapshot:135]: I normally like to fold fixup commits to their associated change, but it isn't particularly obvious to me which caused this.

My guess is 3bfb2b673f (although I don't have the energy to investigate the details). And if someone confirms, I'm happy to fold it in...

**Comment by jsoref** [comment]
@shilman: thanks, everything looks green-ish.

**Review by shilman — APPROVED** [review]
🎉

**Pull request merged**